packages/clerk-js/src/ui/components/OrganizationProfile/__tests__/LeaveOrganizationPage.test.tsx

```diff
@@ -9,8 +9,9 @@ import { LeaveOrganizationForm } from '../ActionConfirmationPage';
 
 const { createFixtures } = bindCreateFixtures('OrganizationProfile');
 
-//TODO-RETHEME
-describe.skip('LeaveOrganizationPage', () => {
+const onSuccessCb = jest.fn();
+const onResetCb = jest.fn();
```

Comment:
🙃 instead of creating those 2 module level variables i would suggest you explicitly use values in the props.
```
onSuccess={ jest.fn() }
onReset={ jest.fn() }
```
Even though in this case it may not cause any issue, in the future it may cause test flakiness since someone may assert those variables and we haven't set any mock resetting or clearing for those.